wolfCLU/src/hash/wolfsslHash.c

```diff
@@ -118,16 +139,23 @@ int wolfsslHash(char* in, char* out, char* alg, int size)
                 /* if outFile exists */
                 for (i = 0; i < size; i++) {
                     /* writes hashed output to outFile */
-                    fprintf(outFile, "%02x", output[i]);
+                    if (outputAsHexString)
+                        fprintf(outFile, "%02x", output[i]);
+                    else
+                        fprintf(outFile, "%c", output[i]);
                 }
                 fclose(outFile);
             }
         }
         else {
+            //printf("output: len %d\n", size);
```

Comment:
Yes and its commented out, but was useful during development. Fine to leave as-is. As I said in commit b92011d it was messing up the piped output to file, so commented it out.